Library/PhotosController.swift

```diff
@@ -48,9 +47,31 @@ class PhotosController: UICollectionViewController {
             self.sections = Photo.constructRemoteElements()
             self.collectionView?.reloadData()
         }
+
+        #if os(tvOS)
+        let playPauseTapRecognizer = UITapGestureRecognizer(target: self, action: #selector(self.playPause(gesture:)))
+        playPauseTapRecognizer.allowedPressTypes = [NSNumber(value: UIPressType.playPause.rawValue)]
```

Comment:
What is this `allowedPressTypes`? What kind of gesture on the collectionview will activate the slideshow? Might be more clear to put the pressType var in a value with a clearer name, before setting it as allowedPressTypes`